web/src/constants/DataStore.constants.ts

```diff
@@ -0,0 +1,15 @@
+import {SupportedDataStores} from '../types/Config.types';
+
+export const SupportedDataStoresToName = {
+  [SupportedDataStores.JAEGER]: 'Jaeger',
+  [SupportedDataStores.OpenSearch]: 'OpenSearch',
+  [SupportedDataStores.SignalFX]: 'SignalFX',
+  [SupportedDataStores.TEMPO]: 'Tempo',
+} as const;
+
+export const SupportedDataStoresToDocsLink = {
+  [SupportedDataStores.JAEGER]: 'https://docs.tracetest.io/run-locally/#installing-jaeger',
```

Comment:
This is still missing, we can come back later on to fix it. CC: @adnanrahic

Reply:
We'll merge docs today. I'll have links ready for you to update this by end of today. 😄